extensions/douban/src/search-books.tsx

```diff
@@ -116,6 +117,11 @@ export default function Command() {
                   icon={Icon.AppWindowList}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${book.title}(豆瓣)](${book.url})`}
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.